plugins/notifications-backend/src/service/handlers.ts

```diff
@@ -144,10 +157,25 @@ export async function getNotifications(
     throw new Error('user parameter is missing in request');
   }
 
+  const orderByIndex = FieldsAllowedToOrderBy.findIndex(
```

Comment:
i think it is easier to read with 'if-else' statements. but i don't mind. E.g.
`if (sorting.fieldName)`{
     // search for value in array
     if (index < 0) {
       throw;
    }
} else {
     // use default
}